pom.xml

```diff
@@ -477,7 +477,7 @@
 
     <properties>
         <!--Identity Governance Version-->
-        <identity.governance.version>1.10.0</identity.governance.version>
+        <identity.governance.version>1.11.4-SNAPSHOT</identity.governance.version>
```

Comment:
The snapshot versions need to be bumped after dependent PRs are merged. The snapshot values will be replaced with new release versions.